owiddata/06.plotVaxCountryData.py

```diff
@@ -92,7 +92,7 @@ def main(args):
     # script after the updated image is committed
     owid_stats['owid_doses_bargraph'] = \
     f'https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/{args.doses_bargraph}'
-    owid_stats['owid_doses_bargraph'] = \
```

Comment:
This is the only actual change. Everything else was just produced by running the pipeline locally.